activity_browser/bwutils/montecarlo.py

```diff
@@ -330,14 +330,14 @@ def perform_MonteCarlo_LCA(project='default', cs_name=None, iterations=10):
     mc = perform_MonteCarlo_LCA(project='ei34', cs_name='kraft paper', iterations=15)
 
     # test the get_results_by() method
-    print('\nTesting the get_results_by() method')
+#    print('\nTesting the get_results_by() method')
```

Comment:
Well for no particular reason really, I didn't put the logger here because it serves no purpose. I could just as well have left these lines the same though, I admit it doesn't make a difference